challenge.rb

```diff
@@ -1,26 +1,34 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map do |val|
+    val.capitalize
```

Comment:
looks good. take a look at Ruby's to_proc method...`&`. you can make this more concise.

http://www.brianstorti.com/understanding-ruby-idiom-map-with-symbol/